src/chess/bitboard.h

```diff
@@ -209,21 +221,25 @@ class Move {
   uint16_t as_nn_index() const;
 
   bool operator==(const Move& other) const {
-    return from_ == other.from_ && to_ == other.to_ &&
-           promotion_ == other.promotion_;
+    return from() == other.from() && to() == other.to() &&
+           promotion() == other.promotion();
   }
 
   bool operator!=(const Move& other) const { return !operator==(other); }
-  operator bool() const { return from_.as_int() != 0 || to_.as_int() != 0; }
+  operator bool() const { return from().as_int() != 0 || to().as_int() != 0; }
 
   void Mirror() {
-    from_.Mirror();
-    to_.Mirror();
+    BoardSquare tmpfrom = from();
+    BoardSquare tmpto = to();
+    tmpfrom.Mirror();
+    tmpto.Mirror();
+    SetFrom(tmpfrom);
+    SetTo(tmpto);
```

Comment:
Wouldn't this be simpler as 
```
SetFrom(from().Mirror());
SetTo(to().Mirror());
```
?

Or am I missing something obvious about that?